File: src/main/java/ua/softgroup/matrix/server/supervisor/consumer/json/SettingsJson.java
package ua.softgroup.matrix.server.supervisor.consumer.json;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

/**
 * @author Oleksandr Tyshkovets <sg.olexander@gmail.com>
 */
public class SettingsJson {

    @JsonProperty
    private boolean success;
    @JsonProperty
    private List<SettingJson> list;

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public List<SettingJson> getList() {
        return list;
    }

    public void setList(List<SettingJson> list) {
        this.list = list;
    }
}

File: src/main/scala/ua/softgroup/matrix/server/supervisor/consumer/json/SettingsJson.scala
package ua.softgroup.matrix.server.supervisor.consumer.json

import java.util

import com.fasterxml.jackson.annotation.JsonProperty

import scala.beans.{BeanProperty, BooleanBeanProperty}

/**
  * @author Oleksandr Tyshkovets <olexandr.tyshkovets@gmail.com>
  */
class SettingsJson {

  @JsonProperty
  @BooleanBeanProperty
  var success: Boolean = _

  @JsonProperty
  @BeanProperty
  var list: List[SettingJson] = _

  override def toString = s"SettingsJson(success=$success, list=$list)"

}
